rpp/browser.py

```diff
@@ -24,6 +24,7 @@ def __init__(self):
         self.path = self.get_path()
         self.name: str = self.get_name()
         self.process: str = self.path.split("\\")[-1]
+        self.microsoft_store: bool = self.is_microsoft_store_app()
```

Comment:
I would make this attribute a property, in the sense that it should have a setter and getter to make it more modular.